test/suite/test_hs_evict_race01.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
```

Comment:
I think this should be a checkpoint test? evict_race is not clear as it doesn't say what it races with. I prefer just eviction test or checkpoint test.

Reply:
I looked the code. It's ok.

Reply:
Could be a test_bug? I like more explicit names though. test_hs_checkpoint_eviction_race?